src/components/PhoneNumberEntry.vue

```diff
@@ -53,22 +78,19 @@ export default {
         },
       ];
     },
-    /** Returns if the phone number entry is not complete */
-    isPhoneNumberNotComplete() {
-      return this.phoneNumber.length < 10;
-    },
+
     /** Updates phone number based on user entry */
-    updatePhoneNumber(event) {
+    updatePhoneNumberEntry(event) {
       if (event.target.value.length == 0) {
         this.invalidPhoneNumberMessage = "";
       } else if (event.target.value.length > 10) {
```

Comment:
maxPhoneNumberLength = 10 we can set a variable

Reply:
It's a phone number. That's the component. So it's always going to be 10.